charts/bulk-scan-payment-processor/values.preview.template.yaml

```diff
@@ -1,6 +1,22 @@
 java:
+  secrets:
+    SB_CONN_STRING:
+      secretRef: servicebus-secret-namespace-{{ .Release.Name }}
+      key: connectionString
   environment:
+    PAYMENTS_QUEUE_READ_CONNECTION_STRING: "$(SB_CONN_STRING);EntityPath=payments"
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   aadIdentityName:
+servicebus:
+  resourceGroup: bulk-scan-aks
+  teamName: "Software Engineering"
```

Comment:
Worth updating to "Bulk Scanning and Printing"

Reply:
it is same in processor. I noted for next version